generator/src/main/java/com/linkedin/pegasus/generator/DataTemplateGeneratorCmdLineApp.java

```diff
@@ -0,0 +1,259 @@
+/*
+   Copyright (c) 2015 LinkedIn Corp.
```

Comment:
This file is a copy of PegasusDataTemplateGenerator, refactored to use CLI arguments.

Reply:
PegasusdataTemplateGenerator was deprecated (in this PR).

Since this is mostly a copy, the correct way to copyright is to keep the date from the original.